functional_tests/scylla_operator/test_functional.py

```diff
@@ -272,3 +274,57 @@ def test_orphaned_services_multi_rack(db_cluster):
 
     db_cluster.wait_for_pods_readiness(pods_to_wait=2, total_pods=len(db_cluster.nodes))
     assert not get_orphaned_services(db_cluster), "Orphaned services were found after decommission"
+
+
+def test_ha_update_spec_while_rollout_restart(db_cluster: ScyllaPodCluster):
+    """
+    Cover the issue https://github.com/scylladb/scylla-operator/issues/410
+    Validate that cluster resources can be updated while the operator StatefulSet is rolling out.
+    - update cluster specification a few time
+    - start rollout restart in parallel with the update
+    - validate that the cluster specification has been updated
+    """
+    to_terminate = threading.Event()
```

Comment:
"to_terminate" sounds confusing to me. var names must be self-explanatory as much as possible.

Reply:
changed